examples/express-ts/src/api.ts

```diff
@@ -109,7 +117,7 @@ export default (router: express.Router) => {
       throw createStatusCodeError(404);
     }
 
-    const pet = await person.$relatedQuery('pets').insert(req.body);
+    const pet = await person.$relatedQuery<Animal>('pets').insert(req.body);
```

Comment:
Did you have to add these because we're using an older version of objection?

Reply:
Depends on how old. Your recent typings change for `$relatedQuery` gets its type from the model its called on. That would yield a Person in this case. We need an Animal.

Reply:
You'd have to make the typings infer the return type from the provided relation name.

!!!!!!